frontend/index_es.html

```diff
@@ -0,0 +1,551 @@
+          <!--Navigation tabs are commented out for reference
+          <li class="nav-item"><a href="#" class="nav-link">Login</a></li>
+          <li class="nav-item"><a href="#" class="nav-link">Help</a></li>
+          -->
+        </ul>
+        <!-- desktop search button -->
+        <button type="button" class="custom-toggler site-navbar-toggler site-searchbar-toggle d-none d-md-inline" data-toggle="collapse" data-target="#searchbar-collapse" aria-expanded="false" aria-label="Toggle Search">
+          <span class="fa fa-search" aria-hidden="true"></span>
+        </button>
+        <!--Translation Button-->
+        <div class="dropdown">
+          <button class="dropbtn">Lenguaje</button>
+          <div class="dropdown-content">
+            <a href="#" onclick="getEnglishPage()">English</a>
+            <a href="#" onclick="">Español</a>
+            <script >
+              function getEnglishPage(){
+                window.location.href = "index.html";
+              }
+            </script>
+          </div>
+        </div>
+      </div>
+
+    </nav>
+    </div><!--container-->
+    </div><!--program-header-->
+    <!-- mobile search button -->
+    <button type="button" class="custom-toggler site-navbar-toggler site-searchbar-toggle d-md-none" data-toggle="collapse" data-target="#searchbar-collapse-mobile" aria-expanded="false" aria-label="Toggle Search">
+    <span class="fa fa-search" aria-hidden="true"></span>
+    </button>   
+    <!-- Start collapsible site search bar -->
+    <div class="site-search-bar">
+    <div class="container">
+    <div class="row">
+      <div class="col">
+        <nav class="navbar navbar-light">
+          <div class="navbar-collapse collapse" id="searchbar-collapse">
+            <form
+                method="get"
+                class="site-searchbar-form form-inline ThunderstoneForm"
+                action="https://search4.nrel.gov/texis/search/" 
+                id="ThunderstoneSearchForm">
+
+              <input type="hidden" name="pr" value="metanrel" />
+              <div class="form-group">
+                  <label for="query" class="sr-only">Buscar NREL.gov</label>
+                  <input
+                      type="search"
+                      id="query" name="query"
+                      maxlength="2047"
+                      placeholder="Buscar NREL.gov"
+                      class="form-control"
+                      aria-label="Buscar NREL.gov">
+                  <button type="submit" class="btn btn-default">Buscar</button>
+              </div>
+            </form>
+          </div>
+        </nav>
+      </div>
+    </div>
+    </div>
+    </div>    
+    <!-- End collapsible site search bar -->
+    <!--startindex-->
+
+		<div class="container" id="content" role="main">
+         <div class="row">
+            <div class="col-lg-9">
+               <h1>Universidad de Puerto Rico Recinto de Mayagüez Proyecto NSF CIVIC</h1>
```

Comment:
again, per https://github.com/e-mission/e-mission-docs/issues/842
"you need to have hardcoded spanish text and use the spanish version of the dynamic config to fill in the dynamic parts of the page"

This is part of the dynamic config. In the English page, this is 
`Thank you for participating in the <a id="studyTitleTwo"></a>`
https://github.com/e-mission/nrel-openpath-join-page/blob/main/frontend/index.html#L196

Reply:
Yes, this is something that we noticed. For the main header, it originally had "<h1 id="studyTitleOne"></h1>", but "studyTitleOne" calls in "$(data.intro.deployment_partner_name)" and "deployment_partner_name" is in English. It has no Spanish version in any of the study config files. We will revert this change, set the dynamic part in Spanish, but since there is no Spanish version of "deployment_partner_name", when using uprm-civic.nrel-op.json file, the header will say "University of Puerto Rico at Mayagüez Proyecto UPRM CIVIC". Notice the Spanglish in the sentence. This is something that will happen to all of the studies.

Reply:
@dcolondavila good point. This is also true for the MassCEC case in the demo. Please expand the existing configs to include english and spanish versions of the deployment partner name. When I originally created the configs, the limited number of use cases were simple enough that one version of the partner name was sufficient; now that it is not, we should expand the config.

Please make sure to edit *all* existing configs.